.github/workflows/deploy.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         id: env_vars
         run: |
           echo "Running on branch ${{ github.ref }}"
-          if [ "${{ github.ref }}" = "refs/heads/master" ]; then
+          if [ "${{ github.ref }}" = "refs/heads/development" ]; then
```

Comment:
same as above.

Reply:
I apologize. It is going to be `development`. I am editing the requirement.